react-mutations/my-app/src/App.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useState } from 'react';
+import { PokemonList } from './PokemonList';
+
+export function App() {
+  const [pokedex, setPokedex] = useState([
+    { number: '001', name: 'Bulbasaur' },
+    { number: '004', name: 'Charmander' },
+    { number: '007', name: 'Squirtle' },
+    { number: '025', name: 'Pikachu' },
+    { number: '039', name: 'Jigglypuff' },
+  ]);
+
+  function handleAdd(): void {
+    const newPoke = {
+      number: '009',
+      name: 'Golden',
+    };
+    setPokedex([...pokedex, newPoke]);
+  }
+
+  function handleUpdate(): void {
+    const updatedPoke = {
+      number: '001',
+      name: 'Foggy',
+    };
+    setPokedex([...pokedex, updatedPoke]);
+  }
+
+  function handleRemove(): void {
+    const deletedItems = [...pokedex];
+    deletedItems.splice(deletedItems.length - 1, 1);
```

Comment:
This is always deleting the last item in the list. You should be able to delete any item. The `.filter` method is the easiest way to do this.